enhancements/declarative-index-config.md

```diff
@@ -549,21 +559,15 @@ Similarly, removing a package directory from `community-operators` and pushing t
 As a user with pull permission from the namespace an index is hosted in/as a component of OLM, I can validate the structure of the content of an index using the json representation.
  
 ```bash
-$ cd community-operators 
-$ sed -i '/defaultChannel/d' etcd/etcd.json //delete the line "defaultChannel":"singlenamespace-alpha"
+$ sed -i '/defaultChannel/d' community-operators/etcd/etcd.json //delete the line "defaultChannel":"singlenamespace-alpha"
```

Comment:
Alternatively, I think the example may be more clear if we start with a "bad" index, validate, then correct it and validate again.

Reply:
According to me removing a required field from a valid config (so that we can see how the configs are invalid) is starting with a bad index